factcast-store-sub-redis/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.factcast</groupId>
+    <artifactId>factcast</artifactId>
+    <version>0.9.5-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <artifactId>factcast-store-sub-redis</artifactId>
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Why not 17? I think the store is at 17 🤔

Reply:
true